nautobot/core/api/views.py

```diff
@@ -208,8 +208,8 @@ def get_serializer(self, *args, **kwargs):
     def get_serializer_context(self):
         context = super().get_serializer_context()
         if "text/csv" in self.request.accepted_media_type:
-            # CSV rendering should always use depth 1
-            context["depth"] = 1
+            # CSV rendering should always use depth 0
+            context["depth"] = 0
```

Comment:
I'm not clear on why this changed. Can you please provide a brief explanation?

Reply:
I suppose it was required in the past so that `CSVParser` could access data from FK fields like "url," "composite_key," and so on. But as we no longer require access to these fields, we no longer need `depth=1`

Before:
```
device,status,name
device_composite_key,Active,Interface
```

After:
```
name,status,device__name,device__location__name,...
Interface One,Active,Device Name,device location name
```